package.json

```diff
@@ -32,7 +35,11 @@
     "concurrently": "^8.2.2",
     "cross-env": "^7.0.3",
     "husky": "^9.0.11",
-    "validate-branch-name": "^1.3.0",
-    "lint-staged": "^15.2.2"
+    "lint-staged": "^15.2.2",
+    "validate-branch-name": "^1.3.0"
+  },
+  "dependencies": {
```

Comment:
These dependencies should be in the client's .json package. It's amazing that it even passed the linter

Reply:
we have the same problem on most of our PR, can we check this together?